CHANGES

```diff
@@ -1,7 +1,9 @@
-Next (YYYY-MM-DD)
------------------
+0.15.0 (2018-01-10)
+-------------------
 - Add api_name and api_version class attributes (#207).
-
+- Add DirectionsMatrix class for access to the Matrix API V1 (#212).
```

Comment:
@perrygeo what do you think about the name? The Web API has `directions-matrix` in its endpoint URL and I followed suit.

Reply:
Directions Matrix sounds good to me. There are a few cases in the docs that we should change for consistency.